src/features/profile/services/profile.service.ts

```diff
@@ -0,0 +1,52 @@
+import { BACKEND_V2_URL } from "app/config";
+import { BackendError } from "app/exceptions";
+import axios from "axios";
+import { User } from "core/users/types";
+
+const URL = `${BACKEND_V2_URL}/profiles`;
+
+export async function fetchOneProfile(
+  token: string,
+  profile_id: number
+): Promise<OneProfileResponse> {
+  try {
+    const response = await axios.get(URL + "/" + profile_id, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    return response.data;
+  } catch (error) {
+    throw new BackendError(error);
+  }
+}
+
+export async function updateProfile(
+  token: string,
+  body: updateProfileBody,
+  profile_id: number
+) {
+  try {
+    const response = await axios.patch(URL + "/" + profile_id, body, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    return response.data;
+  } catch (error) {
+    throw new BackendError(error);
+  }
+}
+
+export interface updateProfileBody {
```

Comment:
CapitalCase creo se llama o PascalCase
```suggestion
export interface UpdateProfileBody {
```
![image](https://github.com/luizzzito/ER-frontend/assets/22270579/8b83689e-0a88-4e5b-8db7-ee4adcb01431)

Reply:
Pascal